src/ui/components/Library/Library.module.css

```diff
@@ -272,3 +274,45 @@
 .dropdownTrigger:focus {
   box-shadow: 0 0 0 var(--ring-width) var(--ring-color);
 }
+
+.testTitle {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.25rem;
+  border-bottom: 1px solid var(--theme-border);
+  padding: 0.93rem 1rem;
+}
+
+.testTitle > div {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  font-size: 1.125rem;
+  font-weight: 500;
+}
+
+.replayRow {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.5rem;
+  border-bottom: 1px solid var(--chrome);
+  padding: 0.625rem 1rem;
+  cursor: pointer;
+  transition: background-color 0.2s;
+}
+
+.replayListTitle {
+  padding: 0.5rem 1rem 0.08rem;
+  font-size: 1.125rem;
```

Comment:
This is another example of a bizarre size increment.

Reply:
See previous resolved comments with Ryan. These are a natural side effect of polishing a design to make things line up. It's not a flaw in the design, and it will continue to happen.